osu.Game.Rulesets.Osu/Difficulty/Evaluators/AimEvaluator.cs

```diff
@@ -34,19 +36,24 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool with
             var osuLastLastObj = (OsuDifficultyHitObject)current.Previous(1);
 
             // Calculate the velocity to the current hitobject, which starts with a base distance / time assuming the last object is a hitcircle.
-            double currVelocity = osuCurrObj.LazyJumpDistance / osuCurrObj.StrainTime;
+            double currVelocity = osuCurrObj.JumpDistance / osuCurrObj.StrainTime;
+
+            // Bandaid to prevent Lost Umbrella from gaining
```

Comment:
I don't really like to imagine this as a "Lost Umbrella bandaid" but rather a change to stop over-rating patterns that appear in this beatmap (and other similar offenders)...

Reply:
In general - this multiplier exists to prevent slideraim from getting pp for sliderjump
So sliderjump bonus will be only for circle aim

Reply:
Yes, I'm implying you should rephrase the comment/remove it